docs/rootstock/setup_regtest.md

```diff
@@ -0,0 +1,154 @@
+3. make a copy of the `.envrc.example` and configure it according to:
+
+    [fill-out-environment-variables](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#fill-out-environment-variables) optimism documentation
+
+    ```shell
+    cd $PATH_RSKSMART/optimism # if not there already
+    make rsk-env # this will also create the configuration in the future
+    ```
+
+      > [!NOTE]
+      > These congfs shell be automated in the near future (//TODO:).
+
+      - additionally, to make the use of the rsk make targets configure following env vars:
+
+    ```shell
+    # path to optimism
+    export PATH_OPSTACK="$PATH_RSKSMART/optimism"
+
+    # path to bedrock contracts
+    export PATH_CONTRACTS="$PATH_OPSTACK/packages/contracts-bedrock"
+
+    # paths to clients
+    export PATH_CLIENT_CONSENSUS="$PATH_OPSTACK/op-node"
+    export PATH_CLIENT_SEQUENCING="$PATH_RSKSMART/op-geth"
+
+    # regtest 33
+    export L1_CHAIN_ID=33
+    ```
+
+      - also change:
+      ```shell
+      export DEPLOYMENT_CONTEXT=regtest
+      ```
+
+    > [!IMPORTANT]
+    > run `direnv allow` to load the environment variables after change.
+
+4. run your favourite **L1 node**:
+
+    - in docker
+
+    ```shell
+    make rsk-regtest-start-log
+    ```
+
+    - from jar
+
+    ```shell
+    #TODO:
+    ```
+
+#### TTY 2 ([start op geth](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#start-op-geth))
+
+5. build the internal packages and run sequencer client with
+
+    ```shell
+    make rsk-build
+    make rsk-sequencer-fresh # this consists of rsk-wallets rsk-config rsk-create2 rsk-deploy rsk-deploy-sync rsk-create-genesis rsk-jwt rsk-sequencer-init rsk-sequencer-run
+    ```
+
+#### TTY 3 ([start op node](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#start-op-node))
+
+6. run consensus client
+
+    ```shell
+    make rsk-consensus-run
+    ```
+
+#### TTY 4 ([start op batcher](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#start-op-batcher))
+
+7. run batcher client
+
+    ```shell
+    make rsk-batcher-run
+    ```
+
+#### TTY 5 ([start op proposer](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#start-op-proposer))
+
+8. run proposer client
+
+    ```shell
+    make rsk-proposer-run
+    ```
+
+---
+
+| :rocket:    | Liftoff, We Have Liftoff|
+|---------------|:----------------------------------------|
+||[get hacking](https://docs.optimism.io/builders/chain-operators/hacks/overview)|
+
+#### TTY 6 test setup
+
+deposit some RBTC using sdk
+
+```
+cd packages/sdk
+npx hardhat deposit-eth --to 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --amount 100 --network regtest --withdraw false;
+```
+
+> [!NOTE]
+> the `withdraw false` is necessary at the moment as withdrawal not yet supported on RSK
+
+and to verify the balance
+
+```
+cast balance 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --rpc-url http://localhost:8545
+```
```

Comment:
Not for us, but for externals I'd also mention that if the received balance is greater than 0, then setup is successful

Reply:
overkill?